examples/metrics/grafana-dashboards/strimzi-cruise-control.json

```diff
@@ -14,7 +14,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "6.3.0"
+      "version": "7.3.7"
```

Comment:
What is the meaning of this field? Does it indicate versions of Grafana the dashboard supports? If yes, why do we need to increase it when we do not seem to change anything in the dashbord?